inc/inventory/asset/camera.class.php

```diff
@@ -79,21 +80,37 @@ protected function itemdeviceAdded(Item_Devices $itemdevice, $val) {
    }
 
    private function handleResolution($itemdevice, $val, $is_video = false) {
-      $resolution = new \ImageResolution();
-      if (!$resolution->getFromDBByCrit(['name' => $val])) {
-         $resolution->add([
-            'name' => $val,
-            'is_video' => $is_video,
-            'is_dynamic' => 1
-         ]);
+      if (!is_array($val)) {
+         $val = [$val];
       }
 
-      $cam_resolutions = new \Item_DeviceCamera_ImageResolution();
-      $cam_resolutions->add([
-         'item_devicecameras_id' => $itemdevice->fields['id'],
-         'imageresolutions_id' => $resolution->fields['id'],
-         'is_dynamic' => 1
-      ]);
+      foreach ($val as $rsl) {
+         if (empty($val)) {
```

Comment:
I guess you used wrong var.
```suggestion
         if (empty($rsl)) {
```